examples/old_format_file_name/setup.cfg

```diff
@@ -1,4 +1,4 @@
 [build_manpage]
 output=example.1
 parser=IGNORED:get_parser
-parser-file=example.py
+parser_file=example.py
```

Comment:
I keep returning to this PR, and tend to merge - but I need a bit more focus time to understand what it means.  This is just a test-case file...
- do we need to suggest a different format for docs?
- do we need to do some migrations in real use-cases?
- do we need code changing?
- what is the priority?
If you have hints there, I'd appreciate help.

Reply:
The support for anything else than the snake_case has been deprecated some time ago and removed in setuptools release 78.0.0: https://setuptools.pypa.io/en/latest/history.html#v78-0-0 Only usages of setup.cfg are affected.
I don't see the usage of this being mentioned anywhere in the readme.
I don't think there is anything more required in this project as users of setuptools should be aware of this change already (or not – depending on the version they use).